library_generation/test/generate_library_unit_tests.sh

```diff
@@ -134,6 +134,18 @@ download_protoc_failed_with_invalid_arch_test() {
   assertEquals 1 $((res))
 }
 
+download_protoc_succeed_with_baked_protoc() {
+  # this mimics a docker container scenario.
+  # if the specified version matches the docker env var, then it will be just
+  # copied from the docker protoc location (also specified in an env var).
+  export DOCKER_PROTOC_LOCATION=$(mktmp -d)
+  mkdir "${DOCKER_PROTOC_LOCATION}/protoc-99.99"
+  export DOCKER_PROTOC_VERSION="99.99"
+  download_protoc "99.99" "linux-x86_64"
+  assertFileOrDirectoryExists "protoc-99.99"
```

Comment:
Could you explain more why a non-existent version, 99.99, of protoc exists?

Reply:
I added an explanation in the test:
https://github.com/googleapis/sdk-platform-java/blob/358be3261328a85f8e5eb91efd49aae1e6bee6f5/library_generation/test/generate_library_unit_tests.sh#L138-L145